extensions/applicationinsights-clickanalytics-js/src/ClickAnalyticsPlugin.ts

```diff
@@ -0,0 +1,78 @@
+/**
+ * @copyright Microsoft 2020
+ */
+
+import {
+    IPlugin, IConfiguration, IAppInsightsCore,
+    BaseTelemetryPlugin, CoreUtils, ITelemetryItem,
+    IProcessTelemetryContext, ITelemetryPluginChain,
+    _InternalMessageId, ICustomProperties, 
+    LoggingSeverity
+} from "@microsoft/applicationinsights-core-js";
+import { IConfig, Util } from "@microsoft/applicationinsights-common";
+import { 
+    IClickAnalyticsConfiguration, IContentHandler, 
+    IAutoCaptureHandler, IPageActionTelemetry 
+} from './Interfaces/Datamodel';
+import {
+    mergeConfig, BehaviorMapValidator,
+    BehaviorValueValidator, BehaviorEnumValidator, _ExtendedInternalMessageId
+    } from './common/Utils';
+import { PageAction } from './events/PageAction';
+import { AutoCaptureHandler } from "./handlers/AutoCaptureHandler";
+import { DomContentHandler } from "./handlers/DomContentHandler";
+export { BehaviorMapValidator, BehaviorValueValidator, BehaviorEnumValidator }
+
+export class ClickAnalyticsPlugin extends BaseTelemetryPlugin {
+    public identifier: string = 'ClickAnalyticsPlugin';
+    public priority: number = 181;
+    public version = '#version#';
+    private _config: IClickAnalyticsConfiguration;
```

Comment:
General nit, use dynamicProto() to hide the private members and reduce the "this" usages, which can't be minified.
You already have dynamicProto() in the package.json and loaded via the tests.html

Reply:
Discussed offline, will address in a follow-up PR